crates/wasmtime/Cargo.toml

```diff
@@ -152,6 +154,7 @@ default = [
   'stack-switching',
   'std',
   'debug',
+  'compile-time-builtins',
```

Comment:
Given the weight of wasm-compose/tempfile would you be ok leaving this feature off-by-default?

Reply:
FWIW, they won't be in our CLI release artifacts because there is not any way to define compile-time builtins via the CLI at the moment.

But all else being equal I'd rather have them enabled by default just because that seems like it is on the path for making things production ready, and I feel that this feature is fairly production ready already. If we disabled everything by default, and required opt-in for all features, disabling it would make sense to me, but we have a bunch of other stuff on by default that isn't as baked as this already is yet (stack switching, guest debugging, core dumps) so disabling this now doesn't really make sense to me.